### ISSUE TITLE
[Android] CarouselView not showing after back navigation

### ISSUE THREAD

**Issue by Phenek** [alexrainman/CarouselView#465]
### Description
Hello, I'm working on multiple implementation of Carousel View.
And I notice that when I'm using multiple CarouselView on differents ContentPages.
Sometime on Android CarouselViews are disappearing 

### Steps to Reproduce

1. Create Multiple CarouselViews in Multiple ContentPage with multiple Content
2. Go to one Content Page then to another, then back to the first one
3. Some CarouselView just disappeared

### Expected Behavior
The CarouselView should not disappear.

### Basic Information
- Xamarin.Forms: 3.2.****
-CarouselView : 5.2.0

**Comment by alexrainman** [comment]
Your project is messed up. Can't do nuget restore.

**Comment by alexrainman** [comment]
I was able to reproduce this and find a fix. It will be included in next release.

**Comment by Nullstr1ng** [comment]
Hi Alex, it's Xamarin 4.0 now, and I this issue still exists. Not sure what the possible solution is. So I headed over to this issue tab and tried reporting but  it seems that there are already reports regarding this issue.

Basically, the problem is, I have a Shell, and one of the pages is using your control. CarouselView shows on first launch without a problem, going to other tab and then going back to the page where the CarouselView is. The control doesn't show up anymore. I added a background color to CarouselView just to know what could be the issue, so I set the CarouselView's background to Red, run the app, now the app has a red background with the Carousel view pages, and then navigated to another page, and then going back. So this time, the pages where gone, but the red background is still there.

So I think the carousel view pages were removed and was not able to add them back .
